src/consumption.cpp

```diff
@@ -161,6 +219,13 @@ edible_rating player::can_eat( const item &food, bool interactive, bool force )
         return INEDIBLE;
     }
 
+    for( const auto &m : food.type->materials ) {
+        if( !m.obj().edible() ) {
```

Comment:
> It would be even better if meds weren't comestibles in any way. It only causes problems.

Agreed. I'm going to mark the `edible` field of materials with a `@todo` and open a separate issue to track migrating comestibles that don't fulfill the above criteria